tests/test_warm_reboot.py

```diff
@@ -780,3 +789,665 @@ def test_swss_port_state_syncup(dvs, testlog):
             assert oper_status == "down"
         else:
             assert oper_status == "up"
+
+
+#############################################################################
+#                                                                           #
+#                        Routing Warm-Reboot Testing                        #
+#                                                                           #
+#############################################################################
+
+
+def clear_logs(dvs):
```

Comment:
>clear_logs [](start = 4, length = 10)

we must not use the this clear log, use marker, this remove all previous logs, difficult to debug for previous test.

Reply:
done